src/client.rs

```diff
@@ -133,7 +154,7 @@ impl Client {
 
     /// ```wait_network_initialized``` waits for the sidecar to initialize the network,
     /// if the sidecar is enabled.
-    pub async fn wait_network_initialized(&self) -> Result<(), Error> {
+    async fn wait_network_initialized(&self) -> Result<(), Error> {
```

Comment:
I can not think of a use-case where one would not call this method at start. Thus I am proposing to include it in `new_and_init` (see above) and making the method itself private.

Reply:
Can't think of a case either! :+1:

Reply:
👍